sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -44,6 +44,7 @@ import {
 } from "../util/errors";
 import { Typed } from "rhea-promise";
 import { max32BitNumber } from "../util/constants";
+import { Buffer } from "buffer";
```

Comment:
Is this being added for browser support?

Reply:
Yes, the build was failing iirc